packages/design-system/components/Dropzone/__storybook__/dropzone.mdx

```diff
@@ -0,0 +1,78 @@
+import { Meta } from '@storybook/addon-docs'
+
+<Meta title="@baseapp-frontend | designSystem/Dropzone/Dropzone" />
```

Comment:
is this path right? `designSystem/Dropzone/Dropzone` @deboracosilveira ? Also maybe we should name it `Dropzone` to reflect the exported name

Reply:
yes, it's. We can also just use @baseapp-frontend | designSystem/Dropzone. What it does is to create a folder in which we can add more components related to dropzones if we have it in the future. On next cop i want to discuss something related to this and we can make a decision.